src/mon/OSDMonitor.cc

```diff
@@ -7942,17 +7774,6 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 	new Monitor::C_Command(mon, op, 0, rs, rdata, get_last_committed() + 1));
       return true;
     }
-  } else if (prefix == "osd thrash") {
-    int64_t num_epochs;
-    cmd_getval(g_ceph_context, cmdmap, "num_epochs", num_epochs, int64_t(0));
-    // thrash_map is a member var
-    thrash_map = num_epochs;
-    ss << "will thrash map for " << thrash_map << " epochs";
-    ret = thrash();
-    err = 0;
-    if (ret)
-      goto update;
-  } else {
     err = -EINVAL;
```

Comment:
might want to put the
```c++
} else {
```
back, otherwise some commands would fail with error of `EINVAL` if they don't `goto reply` or `goto update`.

for instance:

```
ceph osd reweight-by-utilization 110
2016-12-09 04:35:37.347093 7f19eb809700 -1 WARNING: the following dangerous and experimental features are enabled: *
2016-12-09 04:35:37.356792 7f19eb809700 -1 WARNING: the following dangerous and experimental features are enabled: *
Error EINVAL: no change
```

see https://jenkins.ceph.com/job/ceph-pull-requests/15547/consoleFull#842358957c212b007-e891-4176-9ee7-2f60eca393b7。